Doxyfile

```diff
@@ -467,7 +467,7 @@ LOOKUP_CACHE_SIZE      = 0
 # normally produced when WARNINGS is set to YES.
 # The default value is: NO.
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
```

Comment:
This is fine for generating all the documentation you need as a byproduct. However I would advise, you have this set just for the files you modified. So documentation is generated only for the files you modified and not for every file. There should be some documentation on Doxygen document generation that can give you information on how to do this.

Reply:
I see your point. However, since we do not currently have a standardized way of handling Doxygen, I think it will be better to leave it as it is, so we do not have to change which files are included for every branch.